frontend/www/js/omegaup/components/user/Profile.vue

```diff
@@ -18,7 +18,7 @@
       <div class="panel panel-default">
         <div class="panel-heading">
           <h2 class="panel-title">
-            {{ profile.rankinfo.rank &gt; 0 ? `#${profile.rankinfo.rank} - `:
+            {{ profile.rankinfo.ranking &gt; 0 ? `#${profile.rankinfo.ranking} - `:
```

Comment:
```suggestion
            {{ profile.rankinfo.rank &gt; 0 ? `#${profile.rankinfo.rank} - `:
```

puedes probar todos los archivos que cambiaste? este estaba bien con `rank` :S

Reply:
Prove los cambios ejecutando el comando yarn run dev y usando la vista y no me marca ningún error en consola